Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
```

Comment:
"What is a package manager" would be a more common title. This depends on the tone of the document. This could however help avoid the repeated titles (and potential confusions).
This avoids for example the awkward "More about repositories".

Reply:
In this case I can also have a question for the title of 1.2, like `What are dependencies?`